lib/api/kuzzle.js

```diff
@@ -44,11 +44,17 @@ const
   Statistics = require('./core/statistics'),
   TokenManager = require('./core/auth/tokenManager'),
   Validation = require('./core/validation'),
-  InternalEngineBootstrap = require('../services/internalEngine/bootstrap'),
+  InternalBootstrap = require('../services/bootstrap/internalBootstrap'),
   Vault = require('./core/vault'),
   runShutdown = require('../util/shutdown'),
   Logger = require('../util/log');
 
+// Add meaningful stacktrace in Node.js 10
+// @todo must be removed in Node.js 12
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)